src/Plugin.InAppBilling/Plugin.InAppBilling.csproj

```diff
@@ -1,14 +1,14 @@
-﻿<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
+﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
 	<PropertyGroup>
-		<TargetFrameworks>netstandard2.0;MonoAndroid10.0;Xamarin.iOS10;Xamarin.TVOS10;Xamarin.Mac20</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TargetFrameworks);uap10.0.16299;</TargetFrameworks>
+		<TargetFrameworks>netstandard2.0;MonoAndroid10.0;Xamarin.iOS10;Xamarin.TVOS10;Xamarin.Mac20;net6.0-android21.0;net6.0-ios10.0;net6.0-maccatalyst13.1</TargetFrameworks>
```

Comment:
Why are you compiling using old SDKs? Setting `SupportedOSPlatformVersion` should be enough to communicate minimum supported API version.

APIs from newer SDKs simply wouldn't be accessible if would were to compile against an old SDK.

Maybe I'm missing something, but I've never another seen MAUI NuGet that does that. Everyone is targeting latest SDK
Example: https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui.Core/CommunityToolkit.Maui.Core.csproj#L4

Reply:
Yeah, that didn't seem to work when it compiled the nuget. I opened it in nuget package explorerer and it didn't set it. My assumption is because I am using MSBuildSDKExtras

Reply:
Although, I am not positive... I need to figure that part out to be honest... like how does it know how to install.... or idk maybe it isn't an issue?

Reply:
> Yeah, that didn't seem to work when it compiled the nuget.

You can try including the NuGet in the MAUI project where `SupportedOSPlatformVersion` is set, use something from the NuGet, and see, if there're any warnings like this: https://github.com/rotorgames/Rg.Plugins.Popup/pull/718

Reply:
> like how does it know how to install

From what I understand, 
- It compiles an assembly using provided (usually latest SDK), and records `minimum supported SDK`
- It warns in your project if some APIs are not available in some SDKs from the `minimum - targeted` range, so you can add a guard clauses or what not.
- When such NuGet is included in other projects, all it's code is treated as "Safe to run" on rage of SDKs that is selected by the NuGet.
- If SDK rage of the app is bigger, user will also see a warning when using code from the NuGet, that it's not supported on some SDKs (and potentially would fail at runtime)